mkosi/__init__.py

```diff
@@ -176,6 +176,23 @@ def remove_packages(state: MkosiState) -> None:
             die(f"Removing packages is not supported for {state.config.distribution}")
 
 
+def configure_os_release(state: MkosiState) -> None:
+    """Write IMAGE_ID and IMAGE_VERSION to /usr/lib/os-release in the image."""
+    if not state.config.image_id and not state.config.image_version:
+        return
+
+    osrelease = state.root / "usr/lib/os-release"
+
+    if not osrelease.is_file():
+        return
+
+    with osrelease.open("a") as osrel:
+        if state.config.image_id:
+            osrel.write(f'IMAGE_ID="{state.config.image_id}"\n')
+        if state.config.image_version:
+            osrel.write(f'IMAGE_VERSION="{state.config.image_version}"\n')
```

Comment:
I don't think this actually happens in practice, but we should probably still take care to replace existing fields if they exist instead of just appending new one. We have this `read_os_release()` helper that might be useful here.

Also, I figure we should probably just fix both os-release in /usr and in /etc if they exist.

Reply:
Opted for not using that, since we don't need all the parsing, just replacing or adding lines. Also the function no loops over `{usr/lib,etc}/{os,initrd}-release`